src/Core/Managed/PCL/Extensibility/Implementation/Platform/PlatformImplementation.cs

```diff
@@ -35,5 +35,10 @@ public IDebugOutput GetDebugOutput()
             
             return this.debugOutput;
         }
+
+        public string GetEnvironmentVariable(string name)
+        {
+            return null;
```

Comment:
Maybe something in Microsoft.Extensions.Configuration.EnvironmentVariables or Microsoft.Extensions.Configuration could be used.